src/mongo/db/repl/topology_coordinator.h

```diff
@@ -1043,20 +1043,41 @@ class TopologyCoordinator::PingStats {
     /**
      * Gets the number of failures since start() was last called.
      *
-     * This value is incremented by calls to miss(), cleared by calls to start() and
-     * set to the maximum possible value by calls to hit().
+     * This value is incremented by calls to miss() and cleared by calls to start().
+     * 
      */
     int getNumFailuresSinceLastStart() const {
         return _numFailuresSinceLastStart;
     }
 
+    /**
+     * Gets a flag that indicates whether the number of failed heartbeats has
```

Comment:
When describing a function like this, it's a bit redundant to say "Gets a flag...". The fact that a function returns a boolean should be enough information about the nature of the return value. In a comment like this, it can be clearer to just concisely describe the case when the function returns `true`, and when the function returns `false`.